parsers/BR.py

```diff
@@ -1,4 +1,4 @@
-from collections import defaultdict
+from collections.abc import defaultdict
```

Comment:
why these changes? it seems to break tests 

```
parsers/BR.py:1: in <module>
    from collections.abc import defaultdict
E   ImportError: cannot import name 'defaultdict' from 'collections.abc' (/home/circleci/.pyenv/versions/3.10.5/lib/python3.10/collections/abc.py)
```

and I don't see a mention of defaultdict in https://docs.python.org/3.10/library/collections.abc.html

Reply:
I believe it's due to a depreciation in python 3.9, there are warnings about it in the "normal" ci runs. Haven't had time to check it yet though.